roles/network_plugin/calico/templates/kdd-crds.yml.j2

```diff
@@ -210,3 +210,18 @@ spec:
     plural: networksets
     singular: networkset
 {% endif %}
+{% if calico_version is version('v3.14.0', '>=') %}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
CRD under v1beta1 will be removed in 1.19 so please update this to v1 (this will require changes in spec)

Reply:
Does update needs to be done now? 
In any case the whole kdd-crds.yml.j2 will need a rewrite, shouldn't it be done in another PR?

Reply:
Well I pretty much already rewrite it, or at least change every v1beta1 to v1 in [this PR](https://github.com/kubernetes-sigs/kubespray/pull/6245/files) not sure I want to search another one again :P

Reply:
(I'm waiting on kube-ovn to update their crd before backporting and merging, that's why the pr is still in draft)

Reply:
Actually CRD under v1beta1 will not be removed yet, see: https://github.com/kubernetes/kubernetes/issues/82022
It would be nice to get this fix in. @floryut will it break your patch?